xpallets/transaction-fee/src/lib.rs

```diff
@@ -15,43 +15,47 @@ mod types;
 
```

Comment:
![header](https://cdn.jsdelivr.net/gh/rjman-ljm/resources@master/assets/1623987692255-1623987692252.png)
You finish the work, it should be deleted or modified.